### DISCUSSION TITLE
Component Testing Code Coverage

### DISCUSSION THREAD

**Discussion by parky128** [cypress-io/cypress#25131] [Component Testing]
I have been reading about cypress code coverage and in particular [this plugin](https://github.com/cypress-io/code-coverage), what I can't figure out is how to get coverage generated for component tests. Is this supported? If so are there any guides around how to get this working?

**Comment by tkharuk** [comment]
@parky128 you need to install a couple of plugins and configure them.
Basically follow steps described in a blog post by @bahmutov https://glebbahmutov.com/blog/component-code-coverage

**Comment by lmiller1990** [comment]
Are you using webpack or Vite? There's a few example around for webpack (with Create React App, and some other stacks). Not sure on Vite.

What have you tried? Can you please provide more information, then I point you in the right direction.

**Reply by OrbisK** [reply]
https://github.com/iFaxity/vite-plugin-istanbul/issues/61 this plugin seems to work in the near future

**Reply by lmiller1990** [reply]
Seems it works right now - I was having some issues, but some other people got it working alongside the `cypress/code-coverage` plugin.
